.travis.yml

```diff
@@ -51,6 +51,42 @@ env:
 
 matrix:
   exclude:
+    # excludes base on:
+    # https://travis-ci.org/ingenue/hunter/builds/126111928
+
+    # I don't know why gcc doesn't link to libdl although -dl is added
+    # to compile flags.
+    - os: linux
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=default
+    - os: linux
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=gcc-4-8
+
+    # boost compile error (cannot link to OpenSSL)
+    - os: linux
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
+
+    # In this case the final binary is cannot be linked to OpenSSL with the following error message:
+    # /usr/bin/ld: /home/travis/build/ingenue/hunter/_testing/Hunter/_Base/ebe8903/e97794b/3d05f7a/Install/lib/libssl.a(s3_meth.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
+    - os: linux
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=sanitize-thread
+
+    # boost build error on OS X
+    - os: osx
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=clang-libstdcxx
+
+    # XCode builder (???) seg fault...
```

Comment:
It's a Travis glitch, restart helps: https://travis-ci.org/ingenue/hunter/jobs/126111944